### DISCUSSION TITLE
concurrent connections to a backend

### DISCUSSION THREAD

**Discussion by kapitainsky** [rustic-rs/rustic#980] [Q&A] [answered]
Backend is probably rclone nowadays but nevertheless does `rustic` has any configuration option to control number of concurrent connections? Is it the same parameter for both upload/download?  Clearly there is no one fit all setting here - local mechanical HDD might work the best with much less concurrent operations than 1GB Internet connection to AWS...

I am slowly tuning and testing my `rustic` based backup solution and could not find anything in documentation nor `rustic --help`

**Comment by kapitainsky** [comment]
OK it looks that it is all hardcoded without any method to configure but editing source code and building new custom `rustic` version

e.g. https://github.com/rustic-rs/rustic/issues/981

Definitely missing feature - especially for some of us using remotes with enforced and limited number of connections.

**Comment by aawsome** [comment]
Right, this is currently missing..

**Comment by simonsan** [comment, marked as answer]
Issue for it to subscribe to here: https://github.com/rustic-rs/rustic/issues/981
